### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by djzort** [armadito/armadito-glpi#185]
This project looks to have died. Can you confirm?

**Comment by ghost** [comment]
Yup the project is dead . there is no updates in their websites and news about Armadito . It seems like its discontinued
